### PR TITLE
This commit will create get all users endpoint.

### PR DESCRIPTION
**Pull request by sondos-ahmed**

### CODE REVIEW

File: models/users.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectAllUsers = () => {
+  console.log("rows");
```
Comment (emilyb93) [inline models/users.js]: remove console logs

File: models/users.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectAllUsers = () => {
+  console.log("rows");
+
+  return db.query("SELECT * FROM users;").then(({ rows }) => {
+    console.log(rows);
```
Comment (emilyb93) [inline models/users.js]: here too please

File: __tests__/app.test.js [hunk 3]
```diff
@@ -308,8 +308,33 @@ describe("/api/articles/:article_id/comments", () => {
...
+
+        body.users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
```
Comment (emilyb93) [inline __tests__/app.test.js]: We need to do a length check here to make sure this isnt just an empty array and also remove the console.log

**Review by emilyb93 — APPROVED** [review]
small changes but then free to merge

File: __tests__/app.test.js (hunk 3 above)
Comment (sondos-ahmed) [reply __tests__/app.test.js]: Thanks Emily. I am working on that

**Pull request merged**